File: C#/Log Levels.cs
using System;

static class LogLine
{
    public static string Message(string logLine)
    {
        return logLine.Split(':')[1].Trim();
    }

    public static string LogLevel(string logLine)
    {
        return logLine.Split(':')[0].Trim(' ', '[', ']', ':' ).ToLower();
    }

    public static string Reformat(string logLine)
    {
        string[] NewLog = logLine.Split(':');
        return NewLog[1].Trim() + $" ({NewLog[0].Trim(' ', '[', ']', ':' ).ToLower()})";
    }
}

File: Java/Log_Levels.java
public class LogLevels {
    
    public static String message(String logLine) 
    {
        String[] split = logLine.split(":");
        return split[1].trim();
    }

    public static String logLevel(String logLine) 
    {
        String[] split = logLine.split(":");
        return split[0].replaceAll("[\\p{Ps}\\p{Pe}]", "").trim().toLowerCase();
    }

    public static String reformat(String logLine) 
    {
        String[] split = logLine.split(":");
        StringBuilder sb = new StringBuilder();
        sb.append(split[1].trim());
        sb.append(" (");
        sb.append(split[0].replaceAll("[\\p{Ps}\\p{Pe}]", "").trim().toLowerCase());
        sb.append(")");
        return sb.toString();
    }
}
